docs/rules/avoid-dispatching-multiple-actions-sequentially.md

```diff
@@ -1 +1,60 @@
 # avoid-dispatching-multiple-actions-sequentially
+
+> This rule promotes disallows dispatching multiple actions one after another.
```

Comment:
We try to keep this line equal to the [rule's description](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/master/src/rules/avoid-dispatching-multiple-actions-sequentially.ts#L23).

I like your description a lot more, so could you also update the [description of the rule]((https://github.com/timdeschryver/eslint-plugin-ngrx/blob/master/src/rules/avoid-dispatching-multiple-actions-sequentially.ts#L23)) please?
 
```suggestion
> This rule disallows dispatching multiple actions one after another.
```